src/helpers/activitypub/activity.unit.test.ts

```diff
@@ -15,6 +15,25 @@ import {
     buildCreateActivityAndObjectFromPost,
 } from './activity';
 
+vi.mock('@js-temporal/polyfill', async () => {
+    const original = await vi.importActual<
+        typeof import('@js-temporal/polyfill')
+    >('@js-temporal/polyfill');
+    return {
+        Temporal: {
+            ...original.Temporal,
+            Now: {
+                // Return a fixed instant for deterministic testing
+                instant: vi
+                    .fn()
+                    .mockReturnValue(
+                        original.Temporal.Instant.from('2025-01-15T10:30:00Z'),
+                    ),
+            },
+        },
+    };
+});
```

Comment:
I wonder if we could extract this to a helper? I think we have it in a few places now. Something like:

```ts
export const createTemporalMock = (value: string) => {
  return async () => {
    const original = await vi.importActual<typeof import('@js-temporal/polyfill')>('@js-temporal/polyfill');
    return {
      Temporal: {
        ...original.Temporal,
        Now: {
          instant: vi.fn().mockReturnValue(
            original.Temporal.Instant.from(value)
          ),
        },
      },
    };
  };
};  
```

Then use like:

```ts
import { createTemporalMock } from '...blah'

vi.mock('@js-temporal/polyfill', createTemporalMock('2025-01-15T10:30:00Z'));
```

What do you think?

Reply:
Great idea. But now that we have added fallback to `post.publishedAt` we don't need this mock 😊

Reply:
Even better 🎉